.eslintignore

```diff
@@ -1,3 +1,4 @@
 node_modules
+packages
```

Comment:
Yeah, thanks for calling this out.  I agree that it's worthwhile to support both.  I don't think we've done anything yet to interfere with the _ability_ to factor more commonality out of config than would be possible for actually independent packages.  Glad you're looking out for that!